skbio/sequence/_iupac_sequence.py

```diff
@@ -537,6 +539,37 @@ def expand_degenerates(self):
         return (self._to(sequence=''.join(nondegen_seq)) for nondegen_seq in
                 result)
 
+    def to_regex(self):
```

Comment:
Talked with @gregcaporaso and our plan will be to use `0.4.0-dev` as the version string and at release we will set it to the appropriate version number